resources/ui/main_window.ui

```diff
@@ -364,6 +364,26 @@ QListWidget::item::selected {
                </item>
               </layout>
              </item>
+             <item>
+              <layout class="QVBoxLayout" name="verticalLayout_12">
+               <item>
+                <widget class="QLabel" name="label_qq_friendly">
+                 <property name="minimumSize">
+                  <size>
+                   <width>0</width>
+                   <height>42</height>
+                  </size>
+                 </property>
+                 <property name="text">
+                  <string notr="true">human readable query set in python code</string>
```

Comment:
I think the not translatable flag is still missing.

Reply:
Isn't this the notr="true" - this appeared when I cleared the checkbox in QtCreator